tangos/tools/add_simulation.py

```diff
@@ -150,7 +150,8 @@ def add_objects_to_timestep(self, ts, create_class=core.halo.Halo):
                 halos.append(h)
 
         with pt.ExclusiveLock("db_write_lock"):
-            logger.info("Add %d %ss to timestep %r", len(halos), create_class.__name__, ts)
+            if (create_class.__name__ != "Group"):
```

Comment:
What's the motivation for this change?

```suggestion
            if create_class.__name__ != "Group":
```

Reply:
This change should be removed, was put there for debugging purposes.